### ISSUE TITLE
shuffling of test_dataset is not explicitly done

### ISSUE THREAD

**Issue by ashj9** [tensorflow/tensorflow#27796]
**System information**
- TensorFlow version:2.0 
- Doc Link:https://www.tensorflow.org/alpha/tutorials/generative/pix2pix
**Input Pipeline Cell**

**Documentation/Code Issue:**
Cell has the following code:

1. test_dataset = tf.data.Dataset.list_files(PATH+'test/*.jpg')
2. *#shuffling so that for every epoch a different image is generated*
3. *#to predict and display the progress of our model.*
4. train_dataset = train_dataset.shuffle(BUFFER_SIZE)
5. test_dataset = test_dataset.map(load_image_test)
6. test_dataset = test_dataset.batch(1)

From line no 4, it seems we are not shuffling the test_dataset.  Should it have been 
"test_dataset = test_dataset.shuffle(BUFFER_SIZE)" ?

**Comment by ymodak** [comment]
Thanks for trying TF 2.0 alpha. We need not apply shuffle on test data (however you can choose to shuffle test data as well) since it does not affect learning weights. Shuffling training data is good practice in order to overcome learning weight biases.
